### DISCUSSION TITLE
getting unmerged scaled data from dials.scale

### DISCUSSION THREAD

**Discussion by wojdyr** [dials/dials#1670] [Q&A] [answered]
I'm trying to summarize how to output scaled unmerged data from various scaling programs/pipelines:
https://strucbio.biologie.uni-konstanz.de/ccp4wiki/index.php?title=Scaled_unmerged_data_files

I think in case of dials.scale one should use dials.export, but I don't know the details. Could you write me in 1-2 sentences what the user should do to get scaled unmerged MTZ?

Is DUI also providing interface for scaling?

**Comment by rjgildea** [comment, marked as answer]
Exporting a scaled unmerged MTZ is as simple as
```
dials.export scaled.refl scaled.expt
```
as described in the tutorial:
https://dials.github.io/documentation/tutorials/processing_in_detail_betalactamase.html#exporting-to-unmerged-mtz

**Comment by jbeilstenedmands** [comment]
There is also the option to output this directly from `dials.scale`, using the option`unmerged_mtz=` with a filename, which will output this in addition to the usual dials datafiles
e.g. `dials.scale symmetrized.expt symmetrized.refl unmerged_mtz=scaled_unmerged.mtz`

**Comment by dagewa** [comment]
Regarding DUI: yes it can run scaling with `dials.scale`. It appears that the `unmerged_mtz` option is not accessible there, so there is only one way to get a scaled MTZ, which is by running the export step. This does indeed produce an unmerged MTZ.
